app-config-electron/README.md

```diff
@@ -0,0 +1,49 @@
+# App Config Electron
```

Comment:
Could we get pages in `docs` as well?

Reply:
Is there a way to just link the readme?

Reply:
You could make it a symlink I think, though I would write the readme vs docs differently for sure (docs are more meant as guides than intros, ideally)